app/grandchallenge/serving/views.py

```diff
@@ -111,5 +111,18 @@ def serve_component_interface_value(
         .exists()
     ):
         return protected_storage_redirect(name=civ.file.name)
+    elif (
```

Comment:
Instead of adding an extra block couldn't you set `perms=["algorithms.view_job", "evaluation.view_evaluation"]`?

Reply:
No, that doesn't work. The content type of all permissions must be the same: https://django-guardian.readthedocs.io/en/stable/api/guardian.shortcuts.html#get-objects-for-user

Reply:
Ah, OK then!